tests/functional/tests/bundles/base-yoga.yaml

```diff
@@ -143,15 +143,21 @@ applications:
     charm: ch:nova-compute
     channel: yoga/stable
     num_units: 1
-    storage:
-      ephemeral-device: '40G'
     options:
+      options:
+      enable-live-migration: true
+      enable-resize: true
+      encrypt: true
+      force-raw-images: true
+      libvirt-image-backend: rbd
+      migration-auth-type: ssh
+      virt-type: lxc
```

Comment:
I think a good option to try would be to leave the virt-type option as the default (kvm), and instead deploy nova-compute inside an lxd VM (use `virt-type=virtual-machine` as a machine constraint).